client/jetpack-onboarding/summary-next-steps.jsx

```diff
@@ -75,19 +75,19 @@ export default localize(
 
 		return {
 			steps: {
-				JETPACK_CONNECTION: {
+				jetpack_connection: {
```

Comment:
> Should we change jetpack_connection to jetpack-connect (dash instead of underscore, no ...ion) while we're at it?

Yes, sounds sensible - let's do it, preferably in this PR before merging 😉 .

Reply:
b8e8bd5